### PR TITLE
Add arg prefix

### PR DESCRIPTION
**Pull request by ghost**
All ur.urdf.xacro have a **prefix** parameter which is not defined in ur_robot.urdf.xacro and ur_joint_limited.urdf.xacro. I added it to avoid duplicate file for a different prefix only.

Default is "" to stay backwards compatible.

### CODE REVIEW

**Comment by fmauch** [comment]
Yes, this seems reasonable. However, I assume that the models and launch files for the e-Series have similar missing pieces?

**Comment by ghost** [comment]
I have not looked for e-series, but I can if needed

**Comment by Jarris81** [comment]
Will this be merged soon? I need the prefix to control multiple robots

File: README.md [hunk 1]
```diff
@@ -1,130 +1,17 @@
...
-```roslaunch ur5_moveit_config moveit_rviz.launch config:=true```
+## Authors
+* **Dufay Gaëtan** - *Initial work* - [steliaRT](https://github.com/steliaRT)
 
+## License
+This project is licensed under Stelia Aerospace License
 
+## See also
+* [universal_robot](https://github.com/fmauch/universal_robot)
+* [Universal_Robots_ROS_Driver](https://github.com/steliaRT/Universal_Robots_ROS_Driver)
```
Comment (fmauch) [inline README.md:17]: Please do not completely replace the README file.

**Review by fmauch — CHANGES_REQUESTED** [review]
Except for the README this seems legit. However, I rather plan to merge in the changes from upstream/melodic-devel-staging soon, which would counteract this PR.

**Comment by fmauch** [comment]
Closing in favor of #10

**Pull request closed without merging**